rxbinding-kotlin/src/main/kotlin/com/jakewharton/rxbinding/widget/RxAdapterView.kt

```diff
@@ -9,80 +9,84 @@ import rx.functions.Func0
 import rx.functions.Func1
 
 /**
- * Create an observable of the selected position of {@code view}.
+ * Create an observable of the selected position of `view`. If nothing is selected,
+ * [AdapterView.INVALID_POSITION] will be emitted.
```

Comment:
@JakeWharton FYI, my IDE doesn't seem to actually understand this. Not sure if it's a tooling issue or if I'm missing something...